README.md

```diff
@@ -56,13 +56,21 @@ cp ./iam/.env-example.env ./iam/.env
 cp ./schemas/.env-example.env ./schemas/.env
 ```
 
-3. Start iam, app, and ceramic services concurrently:
-
+3. Start iam, app, and ceramic services concurrently:  
+`# remember to create .env files first`
 ```sh
-# remember to create .env files first
+yarn 
```

Comment:
This should normally not be required if you run `lerna bootstrap`.
This command would install the dependencies, but lerna should actually do this instead.